riscv/Cargo.toml

```
@@ -27,3 +27,4 @@ critical-section-single-hart = ["critical-section/restore-state-bool"]
 critical-section = "1.1.2"
 embedded-hal = "1.0.0"
 riscv-pac = { path = "../riscv-pac", version = "0.1.1", default-features = false }
+paste = "1.0.15"
```

Just to follow the alphabet:
```suggestion
paste = "1.0.15"
riscv-pac = { path = "../riscv-pac", version = "0.1.1", default-features = false }
```